build_docs.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
```

Comment:
should we have a `set -x` up here to ensure that an unhandled failed command interrupts the script?